dotenv/Tiltfile

```diff
@@ -1,8 +1,9 @@
-def dotenv(fn=".env", verbose=False, showValues=False):
+def dotenv(fn=".env", verbose=False, showValues=False, showWarningIfMissing=True):
 
     # Check if file exists
     if not os.path.exists(fn):
-        warn("Environment variables file not found: '%s'" % fn)
+        if showWarningIfMissing:
+            warn("Environment variables file not found: '%s'" % fn)
```

Comment:
Should I add a `watch_file()` here to re-trigger a build if the file is created?
The documentation is not clear about how `watch_file()` responds to missing files that are created after Tiltfile evaluation.
https://docs.tilt.dev/api.html#api.watch_file